lib/ansiblelint/utils.py

```diff
@@ -834,11 +838,9 @@ def get_playbooks_and_roles(options=None):
 
     for p in map(Path, files):
 
-        if any(str(p).startswith(file_path) for file_path in options.exclude_paths):
-            continue
-        elif (next((i for i in p.parts if i.endswith('playbooks')), None) or
+        if (next((i for i in p.parts if i.endswith('playbooks')), None) or
```

Comment:
It's easier to read it like this:
```suggestion
        if any(i.endswith('playbooks') for i in p.parts) or
```